src/DocChecks.jl

```diff
@@ -182,15 +184,16 @@ function linkcheck(link::Markdown.Link, doc::Documents.Document)
     if !haskey(doc.internal.locallinks, link)
         local result
         try
-            result = read(`curl -sI $(link.url) --max-time 10`, String)
+            result = read(`curl -sI --proto =http,https,ftp,ftps $(link.url) --max-time 10`, String)
```

Comment:
Unfortunately interpolating into backticks escapes spaces, so it would try to call a program named "curl -sI --proto =http,https,ftp,ftps". I could do this instead:
```julia
Cmd(String[split(CURL_CMD)..., link.url, "--max-time", "10"])
```
which is more verbose but has no duplication. What do you think?

Reply:
Good point, I didn't realize that. How about just locally setting a variable to the full `Cmd` object (``curl_cmd = `curl -sI --proto =http,https,ftp,ftps $(link.url) --max-time 10` ``) and then interpolating the `curl_cmd` into the error strings? Either way is fine though.

Reply:
That's much nicer, I've done that now